actors/evm/src/interpreter/precompiles.rs

```diff
@@ -93,7 +103,95 @@ fn read_right_pad<'a>(input: impl Into<Cow<'a, [u8]>>, len: usize) -> Cow<'a, [u
     input
 }
 
-// https://github.com/ethereum/go-ethereum/blob/25b35c97289a8db4753cdf5ab7f2b306ec71794d/core/vm/contracts.go#L165
+// --- Precompiles ---
+
+fn todo<RT: Runtime>(_: &RT, _: &[u8]) -> PrecompileResult {
+    todo!()
+}
+
+/// Read right padded BE encoded u64 ID address
+/// returns encoded CID or an empty array if actor not found
+fn get_actor_code_cid<RT: Runtime>(rt: &RT, input: &[u8]) -> PrecompileResult {
+    let id_bytes = read_right_pad(input, 8);
```

Comment:
Don't most precompiles just use 256bit values and yolo? Not critical, but could save the user some time.

Reply:
yes. next pr